src/network/networkd-address.c

```diff
@@ -1149,6 +1149,8 @@ int request_process_address(Request *req) {
         if (r < 0)
                 return r;
 
+        req->message_counter = NULL;
```

Comment:
probably worth having at least a one line comment explaining why this is done, taken from the commit message

Reply:
Thanks. Done.